R/AlgorithmHandler.r

```diff
@@ -0,0 +1,143 @@
+library("base64enc")
+library("rjson")
+
+
```

Comment:
nit, don't need 3 lines here. And I would add a space between each of the functions below